python/feathub/processors/flink/job_submitter/flink_session_cluster_job_submitter.py

```diff
@@ -80,20 +80,21 @@ def submit(
             )
             return FlinkSessionClusterJob(None, self._executor)
 
-        native_flink_table = self.flink_processor.flink_table_builder.build(
-            features=features,
-            keys=keys,
-            start_datetime=start_datetime,
-            end_datetime=end_datetime,
-        )
+        with self.flink_processor.flink_table_builder.class_loader:
```

Comment:
Do pyflink users always need to explicitly specify class loader in order to submit Flink jobs? If not, can you explain why we need it here?

Reply:
I added a comment and the related Flink JIRA ticket at the `__init__` method of `FlinkTableBuilder`.